src/storage/src/hummock/local_version_manager.rs

```diff
@@ -764,12 +764,42 @@ impl LocalVersionManager {
                             epoch
                         );
                     }
+
+                    SharedBufferEvent::Clear(notifier) => {
+                        // Cancel and wait for all ongoing flush.
+                        let ongoing_flush_handles: Vec<_> =
+                            epoch_join_handle.drain().flat_map(|e| e.1).collect();
+                        for handle in &ongoing_flush_handles {
+                            handle.abort();
+                        }
+                        if let Err(e) = try_join_all(ongoing_flush_handles).await {
+                            error!("Failed to join flush handle {:?}", e)
+                        }
+
+                        // Clear pending write requests.
+                        pending_write_requests.clear();
```

Comment:
When we call stop_all_actors, before we clear the shared buffer, can we assume that there is no actor writing the shared buffer? If so there should not be any pending write request. We'd better assert that the pending write request is empty.

Reply:
fixed.